packages/diff-sequences/src/__tests__/index.property.test.ts

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import fc from 'fast-check';
+import diff from '../';
+
+const findCommonItems = (a: Array<string>, b: Array<string>): Array<string> => {
+  const array: Array<string> = [];
+  diff(
+    a.length,
+    b.length,
+    (aIndex: number, bIndex: number) => a[aIndex] === b[bIndex],
+    (nCommon: number, aCommon: number) => {
+      for (; nCommon !== 0; nCommon -= 1, aCommon += 1) {
+        array.push(a[aCommon]);
+      }
+    },
+  );
+  return array;
+};
+
+const extractCount = (data: Array<string>): Map<string, number> => {
+  const countPerChar = new Map<string, number>();
+  for (const item of data) {
+    const currentCount = countPerChar.get(item) || 0;
+    countPerChar.set(item, currentCount + 1);
+  }
+  return countPerChar;
+};
+
+const flatten = (data: Array<Array<string>>) => {
+  const array: Array<string> = [];
+  for (const items of data) {
+    array.push(...items);
+  }
+  return array;
+};
+
+const isSubsequenceOf = (
+  subsequence: Array<string>,
+  sequence: Array<string>,
+): boolean => {
+  let iSub = 0;
+  let iSeq = 0;
+
+  while (iSub !== subsequence.length && iSeq !== sequence.length) {
```

Comment:
@dubzzz Uh oh my bad, is `travis-ci` in an infinite loop because the loop should be:

```js
  for (let iSeq = 0; iSeq !== sequence.length; iSeq += 1) {
    if (iSub !== subsequence.length && subsequence[iSub] === sequence[iSeq]) {
      iSub += 1;
    }
  }
```